features/guidance/turn-lanes.feature

```diff
@@ -599,22 +599,21 @@ Feature: Turn Lane Guidance
     Scenario: Segregated Intersection Merges With Lanes
         Given the node map
             """
-                        f
-
-            e     d
-                      c g
-            a     b
-
-                      h
+                 a e
+                 | |
+                 | |
+                 b d
+            h     c
+             ' -- g - - f
             """
 
         And the ways
-            | nodes | name     | turn:lanes:forward                 | oneway | highway   |
-            | abc   | road     | left\|left\|left\|through\|through | yes    | primary   |
-            | cde   | road     |                                    | yes    | primary   |
-            | hc    | cross    |                                    | yes    | secondary |
-            | cg    | straight |                                    | no     | tertiary  |
-            | cf    | left     |                                    | yes    | primary   |
+            | nodes | name     | turn:lanes:forward                 | oneway | highway   | lanes |
+            | abc   | road     | left\|left\|left\|through\|through | yes    | primary   | 5     |
```

Comment:
So far we do not do that. I've tested it on Berlin, and Berlin was adding 3 tags. It seems that when lanes are tagged for indications, they often come with number of lanes as well. If it turns out to be way different on the planet, we could think about it.